packages/attestation-service/src/db.ts

```diff
@@ -0,0 +1,22 @@
+import { ContractKit, newKit } from '@celo/contractkit'
+import { Sequelize } from 'sequelize'
+import { fetchEnv } from './env'
+
+export let sequelize: Sequelize | undefined
+
+export function initializeDB() {
+  if (sequelize === undefined) {
+    sequelize = new Sequelize(fetchEnv('DB_URL'))
+    return sequelize.authenticate() as Promise<void>
+  }
+
+  return Promise.resolve()
+}
+
+export let kit: ContractKit
+
+export function initializeKit() {
```

Comment:
does this belong here?

Reply:
It may not be perfect, but it felt wrong to me to create a separate file for it. In some ways the blockchain is of course also a DB. But don't feel strongly, where do you think it should be?